cura/CuraApplication.py

```diff
@@ -674,22 +668,7 @@ def getGlobalContainerStack(self) -> Optional["GlobalStack"]:
     def setGlobalContainerStack(self, stack: Optional["GlobalStack"]) -> None:
         self._setLoadingHint(self._i18n_catalog.i18nc("@info:progress", "Initializing Active Machine..."))
         super().setGlobalContainerStack(stack)
-
-    showMessageBox = pyqtSignal(str,str, str, str, int, int,
-                                arguments = ["title", "text", "informativeText", "detailedText","buttons", "icon"])
-    """A reusable dialogbox"""
-
-    def messageBox(self, title, text,
```

Comment:
Removing this is a bit of an API break. Should we keep it in? Maybe deprecate it? I think it's quite likely that existing plug-ins use this feature.